chat-plugins/info.js

```diff
@@ -1065,7 +1065,13 @@ var commands = exports.commands = {
 	weakness: function (target, room, user) {
 		if (!target) return this.parse('/help weakness');
 		if (!this.canBroadcast()) return;
-		var targets = target.split(/[ ,\/]/);
```

Comment:
I'm not sure I like that as much because it lets users enter random stupid things that still work like `fire ,/ //, /, grass` which would still run the command.